PROTO_tests/tests/series_test.py

```diff
@@ -0,0 +1,150 @@
+import pandas as pd
+import pytest
+from pandas.testing import assert_frame_equal
+
+import arkouda as ak
+
+
+class TestSeries:
+    def test_series_creation(self):
+        idx = ak.arange(3)
+        for val in ak.arange(3), ak.array(["A", "B", "C"]):
+            ans = ak.Series(data=val, index=idx).to_list()
+            for series in (
+                ak.Series(data=val, index=idx),
+                ak.Series(data=val),
+                ak.Series(val, idx),
+                ak.Series(val),
+                ak.Series((idx, val)),
+            ):
+                assert isinstance(series, ak.Series)
+                assert isinstance(series.index, ak.Index)
+                assert series.to_list() == ans
+
+        with pytest.raises(TypeError):
+            ak.Series(index=idx)
+
+        with pytest.raises(TypeError):
+            ak.Series((ak.arange(3),))
+
+        with pytest.raises(TypeError):
+            ak.Series()
+
+        with pytest.raises(ValueError):
+            ak.Series(data=ak.arange(3), index=ak.arange(6))
+
+    def test_lookup(self):
```

Comment:
Again, we should parameterize on the allowed `DTYPE` values and handle `Categorical` as a special case of `ak.str_`.

Reply:
Updated

Reply:
Added a new comment, but this only tests different `dtypes` on values now. It still needs to test different `dtypes` for the index.